src/pages/PrintOnlyScreen.tsx

```diff
@@ -63,7 +62,7 @@ const PrintOnlyScreen = ({
   const printerTimer = useRef(0)
   const [okToPrint, setOkToPrint] = useState(true)
   const [isPrinted, updateIsPrinted] = useState(false)
-  const isCardVotesEmpty = isEmptyObject(votes)
+  const isCardVotesEmpty = !votes
```

Comment:
Won't this always be false? 🤔 

![image](https://user-images.githubusercontent.com/28444/73796104-31bb9c00-4761-11ea-887d-289ea44a40d8.png)

Reply:
This is catching the case where `votes` is `undefined`. The big change in this PR is differentiating between `undefined` which means a card that hasn't been used, and `{}` which is a blank ballot.

Reply:
```
const isCardVotesEmpty = votes === undefined
```
?